Scripts/script.js

```diff
@@ -1,56 +1,57 @@
 (function(module) {
-
+  var names = {};
+  names.all = [];
     //project Constructor function
   function Project (project) {
     Object.keys(project).forEach(function(property, index, keys) {
       this[property] = project[property];
     },this);
   }
-  Project.projectList = [];
   //code to create a method to check localStorage for json object and prepend it to the dom
   Project.fetchAll = function() {
+    Project.projectList = [];
     var dataList;
     if (localStorage.storageProjects) {
       dataList = JSON.parse(localStorage.storageProjects);
+      dataList[0].projectTitle = names.all[0];
+      dataList[1].projectTitle = names.all[1];
       Project.projectList = dataList.map(function(project) {
         $('.projects').prepend(new Project(project).toHtml());
       });
-      console.log(Project.projectList);
+
     } else {
       $.getJSON('Data/projectData.json', function(data) {
         localStorage.setItem('storageProjects', JSON.stringify(data));
         dataList = JSON.parse(localStorage.storageProjects);
+        dataList[0].projectTitle = names.all[0];
+        dataList[1].projectTitle = names.all[1];
         Project.projectList = dataList.map(function(project) {
           $('.projects').prepend(new Project(project).toHtml());
         });
       });
     }
   };
+  //get titles from github for projects, probably not going to include in final website build
+  Project.getGithubInfoAndRunFetchAll = function() {
+    $.ajax({
+       url: 'https://api.github.com/users/chadob/repos' +
+       '?er_page=4&sort=updated',
+       type: 'GET',
+       headers: { 'Authorization': 'token ' + githubToken },
+       success: function(data, message, xhr) {
+         names.all.push(data[4].name);
+         names.all.push(data[5].name);
```

Comment:
Again, best to work with dynamic data rather than hardcoding index positions like this.